code-challenges.test.js

```diff
@@ -22,29 +28,71 @@ var num2 = 0
 var num3 = -7
 // Expected output: "-7 is not divisible by three"
 
+describe('isDivisibleByThree', () => {
+  it('should be divisible by three when given 15', () => {
+   expect(isDivisibleByThree(15)).toEqual("15 is divisble by three")
+  })
+  it('should be divisible by three when given 0', () => {
+   expect(isDivisibleByThree(0)).toEqual("0 is divisble by three")
+  })
+  it('should NOT be divisible by three when given -7', () => {
+   expect(isDivisibleByThree(-7)).toEqual("-7 is not divisible by three")
+  })
+})
```

Comment:
Looks great! To improve the workflow, you can bring your test variables into your describe block and reference the variables in your expect statement.
`expect(isDivisibleByThree(num1)).toEqual("15 is divisble by three")`